Makefile

```diff
@@ -35,6 +35,11 @@ test:
 	@echo "  >  \033[32mRunning tests...\033[0m "
 	go test ./... -v
 
+## test: Runs `go test -race` on project test files.
+test-race:
+	@echo "  >  \033[32mRunning race tests...\033[0m "
+	go test ./state/... -race
+
```

Comment:
lets update this to something like `test-state-race`

Reply:
sure